src/Reflex/Todo.hs

```diff
@@ -1,19 +1,30 @@
 {-# LANGUAGE RecursiveDo, ScopedTypeVariables, FlexibleContexts, TypeFamilies, ConstraintKinds, TemplateHaskell #-}
+module Reflex.Todo where
 
 import Prelude hiding (mapM, mapM_, all, sequence)
 
+import Data.Map (Map)
+import qualified Data.Map as Map
 import Reflex
 import Reflex.Dom
 
+insertNew_ :: (Enum k, Ord k) => v -> Map k v -> Map k v
+insertNew_ v m = case Map.maxViewWithKey m of
```

Comment:
Why the `_` suffix here? Is this actually ignoring the return value somehow? Is this a haskell-ism that I'm not familiar with? I always associate it with functions like `mapM_`.